### ISSUE TITLE
[11.0] oe install error l10n_mx_edi

### ISSUE THREAD

**Issue by aguzman22** [odoo/odoo#26891]
Impacted versions:
11.0 enterprise

Steps to reproduce:
Install module l10n_mx_edi
Current behavior:
Exception during JSON request handling.
```
Traceback (most recent call last):
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/addons/l10n_mx_edi/hooks.py", line 58, in _load_xsd_files
    res = objectify.fromstring(response.content)
  File "src/lxml/objectify.pyx", line 1801, in lxml.objectify.fromstring
  File "src/lxml/etree.pyx", line 3213, in lxml.etree.fromstring
  File "src/lxml/parser.pxi", line 1876, in lxml.etree._parseMemoryDocument
  File "src/lxml/parser.pxi", line 1764, in lxml.etree._parseDoc
  File "src/lxml/parser.pxi", line 1126, in lxml.etree._BaseParser._parseDoc
  File "src/lxml/parser.pxi", line 600, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 710, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 639, in lxml.etree._raiseParseError
  File "<string>", line 1
lxml.etree.XMLSyntaxError: Start tag expected, '<' not found, line 1, column 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/http.py", line 650, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/http.py", line 310, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/http.py", line 692, in dispatch
    result = self._call_function(**self.params)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/http.py", line 342, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/http.py", line 335, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/http.py", line 936, in __call__
    return self.method(*args, **kw)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/http.py", line 515, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/addons/web/controllers/main.py", line 938, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/addons/web/controllers/main.py", line 926, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "<decorator-gen-42>", line 2, in button_immediate_install
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/addons/base/module/module.py", line 71, in check_and_log
    return method(self, *args, **kwargs)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/addons/base/module/module.py", line 449, in button_immediate_install
    return self._button_immediate_function(type(self).button_install)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/addons/base/module/module.py", line 542, in _button_immediate_function
    modules.registry.Registry.new(self._cr.dbname, update_module=True)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/modules/registry.py", line 85, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/modules/loading.py", line 380, in load_modules
    loaded_modules, update_module, models_to_check)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/modules/loading.py", line 274, in load_marked_modules
    perform_checks=perform_checks, models_to_check=models_to_check
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/modules/loading.py", line 200, in load_module_graph
    getattr(py_module, post_init)(cr, registry)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/addons/l10n_mx_edi/hooks.py", line 20, in post_init_hook
    _load_xsd_files(cr, registry, url)
  File "/home/odoo/runbot/runbot/runbot/static/build/371303-11-0-9f6a03/odoo/addons/l10n_mx_edi/hooks.py", line 61, in _load_xsd_files
    'You are trying to load an invalid xsd file.\n%s', e.message)
AttributeError: 'XMLSyntaxError' object has no attribute 'message'
```
Expected behavior:
full installation
Video/Screenshot link (optional):`

**Comment by ernesto-telematel** [comment]
@moylop260 FYI

**Comment by mailynC** [comment]
Hello, I'm having this exception too when I try to install l10n_mx_edi

**Comment by mailynC** [comment]
I was able to install the module, but now I'm getting this error when I try to validate an invoice :

`attribute decl. 'TipoRelacion', attribute 'type': The QName value '{http://www.sat.gob.mx/sitio_internet/cfd/catalogos}c_TipoRelacion' does not resolve to a(n) simple type definition., line 36 `

This is a common issue when you import a database, but in this case is a new install, and the workaround explained [here](https://www.odoo.com/documentation/user/11.0/es/accounting/localizations/mexico.html )  is not working

**Comment by mart-e** [comment]
Hello,

I believe this was fixed at odoo/enterprise@2a92c30d6f7a74b62f40f3521a9da947080e41f2
Thanks for the report
